crates/re_edit_ui/src/range1d.rs

```diff
@@ -14,13 +14,15 @@ pub fn edit_range1d(
     ui.label("Min:");
     let response_min = ui.add(
         egui::DragValue::new(min)
-            .clamp_range(f64::NEG_INFINITY..=*max)
+            .clamp_to_range(true)
```

Comment:
I think you want `.clamp_to_range(false)` here and everywhere

Reply:
yeah I noticed while testing again, already fixed
I had this correct on my mac when I tested the egui branch but apparently didn't push this. And until now I was too lazy to put the mac out of the backback again so I instead redid the change, but incorrectly lol